### PR TITLE
`inputOnly` property added

### PR DESCRIPTION
**Pull request by neotob**
- `inputOnly` property added

### CODE REVIEW

File: src/lib/FormGroupLayout.tsx [hunk 1]
```diff
@@ -24,7 +24,16 @@ const FormGroupLayout = <T extends FieldValues>(props: FormGroupLayoutProps<T>)
   const switchLayout = layout === "switch";
   const checkboxLayout = layout === "checkbox";
 
-  return (
+  if (inputOnly && (switchLayout || checkboxLayout)) {
+    throw "'inputOnly' is not possible with switches or checkboxes";
```
Comment (neotrow) [inline src/lib/FormGroupLayout.tsx:28]: very nice 👏

File: cypress/cypress/component/Input/Select.cy.tsx [hunk 2]
```diff
@@ -68,3 +68,27 @@ it("is disabled", () => {
 
   cy.get(`select[name=${name}]`).should("be.disabled");
 });
+
+it("select only works", () => {
```
Comment (neotrow) [inline cypress/cypress/component/Input/Select.cy.tsx:72]: Suggestion but not required:

Maybe move it to a new test file called `InputOnly.cy.tsx`

**Review by neotrow — CHANGES_REQUESTED** [review]

File: cypress/cypress/component/Input/Select.cy.tsx (hunk 2 above)
Comment (neotob) [reply cypress/cypress/component/Input/Select.cy.tsx]: moved it and added some more, and sorry for resolving it.. wrong button :D

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -59,3 +59,13 @@ jobs:
...
         working-directory: ./cypress
+
+      # Upload screenshots/videos
+      - name: Upload snapshots
+        uses: actions/upload-artifact@v3
+        if: failure()
+        with:
+          name: cypress-snapshots-${{ matrix.node-version }}
+          path: |
+            /home/runner/work/react-hook-form-components/react-hook-form-components/cypress/cypress/screenshots
```
[the remark this answers is not in the document]
Comment (neotob) [reply .github/workflows/ci.yml]: it seems to work, but i did not have any screenshots, only videos.. now im not sure if that is on purpose or not.. im trying now with a failed test again

File: src/lib/FormGroupLayout.tsx (hunk 1 above)
Comment (neotob) [reply src/lib/FormGroupLayout.tsx:28]: i would say it doesn't really makes sense to have that right? at least from our design its not that easy possible...

File: .github/workflows/ci.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (neotob) [reply .github/workflows/ci.yml]: ok screenshots are only taken if a test failed, so everything is fine!

File: cypress/cypress/component/Input/Select.cy.tsx (hunk 2 above)
Comment (neotrow) [reply cypress/cypress/component/Input/Select.cy.tsx]: no worries and thanks!

File: .github/workflows/ci.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (neotrow) [reply .github/workflows/ci.yml]: thanks!

File: src/lib/FormGroupLayout.tsx (hunk 1 above)
Comment (neotrow) [reply src/lib/FormGroupLayout.tsx:28]: yes, perfect like this :-)

**Review by neotrow — APPROVED** [review]

**Pull request merged**